src/drivers/mct/cime_config/config_component.xml

```diff
@@ -1947,7 +1960,8 @@
     </values>
     <group>mach_pes</group>
     <file>env_mach_pes.xml</file>
-    <desc>Number of instances for each component</desc>
+    <desc>Number of instances for each component.  If MULTI_COUPLER is True
+    only NINST_CPL is used, if MULTI_COUPLER is False NINST_CPL is 1.</desc>
```

Comment:
After "only NINST_CPL is used" I would add something like, "and all components have this many instances", then change the following comma to a semi-colon. So the full description will read:

```
Number of instances for each component. If MULTI_COUPLER is True, only NINST_CPL is used and all components have this many instances; if MULTI_COUPLER is False, NINST_CPL is 1.
```